src/main/kotlin/io/github/mojira/arisa/modules/ResolveTrashModule.kt

```diff
@@ -0,0 +1,18 @@
+package io.github.mojira.arisa.modules
+
+import arrow.core.Either
+import arrow.core.extensions.fx
+
+class ResolveTrashModule() : Module<ResolveTrashModule.Request> {
+    data class Request(
+        val projectKey: String,
+        val resolveAsInvalid: () -> Either<Throwable, Unit>
+    )
+
+    override fun invoke(request: Request): Either<ModuleError, ModuleResponse> = with(request) {
+        Either.fx {
+            assertEquals(projectKey, "TRASH").bind()
```

Comment:
I wonder if it would make sense to move the whitelist to the modules and add an assertIsWhitelisted or something.